### PR TITLE
Collections: Frequence task

### PR DESCRIPTION
**Pull request by denisdimitrov**

### CODE REVIEW

File: src/frequence/Frequence.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            if (freqChar.get(each).equals(maxValue)) {
+                return each;
+            }
+        }
+        Optional<Integer> possible = Optional.of(maxValue);
+        possible.isPresent();
+        return null;
+    }
+
+    public Map<String, Integer> wordFrequence(String[] text) {
```
Comment (EmilGeorgiev) [inline src/frequence/Frequence.java]: pass as parameter ```String text``` and than split this text by words.

Comment (denisdimitrov) [inline src/frequence/Frequence.java]: Okey.

**Comment by denisdimitrov** [comment]
Done.

File: src/frequence/Frequence.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * @author Denis Dimitrov(@denis.k.dimitrov@gmail.com)
+ */
+public class Frequence {
+    private final String text;
+
+    public Frequence(String text) {
+        this.text = text;
+    }
+
+    public Map<Character, Integer> charFrequence() {
```
Comment (EmilGeorgiev) [inline src/frequence/Frequence.java]: if  this methos is called only in method ```mostFrequentCharacter``` it  should be private method.

File: src/frequence/Frequence.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        Map<Character, Integer> freqChar = charFrequence();
+        Integer maxValue = Collections.max(freqChar.values());
+        for (Character each : freqChar.keySet()) {
+            if (freqChar.get(each).equals(maxValue)) {
+                return each;
+            }
+        }
+        Optional<Integer> possible = Optional.of(maxValue);
+        possible.isPresent();
+        return null;
```
Comment (EmilGeorgiev) [inline src/frequence/Frequence.java]: return Optional.of(maxValue)

Comment (denisdimitrov) [inline src/frequence/Frequence.java]: I was not supposed to commit that, I was just playing and experimenting with the Optional. I will delete it.

**Comment by denisdimitrov** [comment]
Method changed to private.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**